plugins/woocommerce/includes/admin/class-wc-admin-assets.php

```diff
@@ -124,44 +147,204 @@ public function admin_styles() {
 			}
 		}
 
+		/**
+		 * Get the scripts used for registration.
+		 *
+		 * @return array
+		 */
+		private function get_scripts() {
+			$suffix     = Constants::is_true( 'SCRIPT_DEBUG' ) ? '' : '.min';
+			$version    = Constants::get_constant( 'WC_VERSION' );
+			$plugin_url = WC()->plugin_url();
+
+			return array(
+				array(
+					'handle'       => 'woocommerce_admin',
+					'path'         => $plugin_url . '/assets/js/admin/woocommerce_admin' . $suffix . '.js',
+					'dependencies' => array( 'jquery', 'wc-jquery-blockui', 'jquery-ui-sortable', 'jquery-ui-widget', 'jquery-ui-core', 'wc-jquery-tiptip' ),
+					'version'      => $version,
+				),
+				array(
+					'legacy_handle' => 'jquery-blockui',
+					'handle'        => 'wc-jquery-blockui',
+					'path'          => $plugin_url . '/assets/js/jquery-blockui/jquery.blockUI' . $suffix . '.js',
+					'dependencies'  => array( 'jquery' ),
+					'version'       => '2.70',
+					'in_footer'     => true,
+				),
+				array(
+					'legacy_handle' => 'jquery-tiptip',
+					'handle'        => 'wc-jquery-tiptip',
+					'path'          => $plugin_url . '/assets/js/jquery-tiptip/jquery.tipTip' . $suffix . '.js',
+					'dependencies'  => array( 'jquery', 'dompurify' ),
+					'version'       => $version,
+					'in_footer'     => true,
+				),
+				array(
+					'legacy_handle' => 'round',
+					'handle'        => 'wc-round',
+					'path'          => $plugin_url . '/assets/js/round/round' . $suffix . '.js',
+					'dependencies'  => array( 'jquery' ),
+					'version'       => $version,
+				),
+				array(
+					'handle'       => 'wc-admin-meta-boxes',
+					'path'         => $plugin_url . '/assets/js/admin/meta-boxes' . $suffix . '.js',
+					'dependencies' => array( 'jquery', 'jquery-ui-datepicker', 'jquery-ui-sortable', 'wc-accounting', 'wc-round', 'wc-enhanced-select', 'plupload-all', 'wc-stupidtable', 'wc-jquery-tiptip', 'wc-jquery-blockui' ),
```

Comment:
Comparing with original `array( 'jquery', 'jquery-ui-datepicker', 'jquery-ui-sortable', 'accounting', 'round', 'wc-enhanced-select', 'plupload-all', 'stupidtable', 'jquery-tiptip' )` we added `wc-jquery-blockui` to deps. Is it intended?